packages/aws-cdk-lib/aws-apigateway/lib/method.ts

```diff
@@ -303,6 +314,19 @@ export class Method extends Resource {
     this.methodResponses.push(methodResponse);
   }
 
+  private getMethodAuthorizationType(options: MethodOptions, defaultMethodOptions: MethodOptions, authorizerType?: AuthorizationType): string {
```

Comment:
add some comment to this method to explain the parameters,  to make it easier for the team to use it in future

Reply:
also, I believe it is better to extract the authorizerType in this method instead of accepting it as a parameter. The logic of extracting this value is also confusing

Reply:
Updated